src/audio/codec_adapter/codec_adapter.c

```diff
@@ -338,37 +338,60 @@ codec_adapter_copy_from_lib_to_sink(const struct codec_processing_data *cpd,
 			 (char *)cpd->out_buff + head_size, tail_size);
 }
 
+/**
+ * \brief Generate zero samples of "bytes" size for the sink.
+ * \param[in] sink - a pointer to sink buffer.
+ * \param[in] bytes - number of zero bytes to produce.
+ *
+ * \return: none.
+ */
+static void generate_zeroes(struct comp_buffer *sink, uint32_t bytes)
```

Comment:
This looks like a useful buffer utility function, so maybe better part of the buffer API. Also what happens if bytes is too big ?